### DISCUSSION TITLE
What is the best Chrome extension that can summarize website text?

### DISCUSSION THREAD

**Discussion by heladasd** [community/community#69157] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

What is the best Chrome extension that can summarize website text?

**Comment by ghost** [comment]
ChatGPT Suite - Resumir tudo!

**Reply by heladasd** [reply]
Thank you very much.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
